src/lib/bootloader/serial_console.rb

```diff
@@ -0,0 +1,70 @@
+require "yast"
+
+Yast.import "Arch"
+
+module Bootloader
+  class SerialConsole
+    PARITY_MAP = {
+      "n" => "no",
+      "o" => "odd",
+      "e" => "even"
+    }
+    SPEED_DEFAULT = 9600
+    PARITY_DEFAULT = "no"
+    WORD_DEFAULT = ""
+
+    def self.load_from_kernel_args(kernel_params)
+      console_parameter = kernel_params.parameter("console")
+      return nil unless console_parameter
+
+      console_parameter = [console_parameter] unless console_parameter.is_a? ::Array
```

Comment:
I know about it, but some people find it tricky and with worse readability. If only experience rubist read code, I can use it :)